src/components/ProgressBar/index.tsx

```diff
@@ -17,15 +18,15 @@ const shineAnimation = keyframes`
 export const progressBarVariants = ['primary', 'success', 'warning', 'info']
 
 const StyledBox = styled(Box, {
-  shouldForwardProp: prop => !['backgroundColor'].includes(prop),
-})`
+  shouldForwardProp: prop => !['backgroundColor'].includes(prop.toString()),
+})<{ backgroundColor: string }>`
```

Comment:
Why not type `backgroundColor` as `Color` all the way down ?

```suggestion
})<{ backgroundColor?: Color }>`
```